### PR TITLE
Feature: add useContext on Context API

### PR DESCRIPTION
**Pull request by jrson83**
### Reference Issues/PRs

None

### What does this implement/fix?

Added `useContext` on Context API to provide better developer experience.
As alternative for `MyContext.get()` you can now use `useContext(MyContext)`.

### CODE REVIEW

File: src/context.ts [hunk 1]
```diff
@@ -12,3 +12,10 @@ export const createContext = (ctx: any) => {
     set: (ctx: any) => (_ctx = ctx)
   }
 }
+
+export const useContext = (ctx: any) => {
+  let _ctx = ctx
+  if (_ctx && typeof(_ctx.get) == "function") {
```
Comment (yandeu) [inline src/context.ts:18]: - I prefer Strict Equality Comparison (===)
- `"function"` should be in single quotes

**Review by yandeu — COMMENTED** [review]
Would be great to have a `context.ssr.test.tsx` test.

File: src/context.ts (hunk 1 above)
Comment (jrson83) [reply src/context.ts]: Done.

**Comment by jrson83** [comment]
> Would be great to have a `context.ssr.test.tsx` test.

Done :blush:

**Pull request merged**